packages/plugin-core/src/components/lookup/utils.ts

```diff
@@ -678,26 +676,10 @@ export class PickerUtilsV2 {
       })
     );
 
-    PickerUtilsV2.refreshLookupView({ buttons: opts.quickpick.buttons });
-  }
-
-  static refreshLookupView(opts: { buttons: DendronBtn[] }) {
-    const { buttons } = opts;
-    const payload: LookupModifierStatePayload = buttons.map(
-      (button: DendronBtn) => {
-        return {
-          type: button.type,
-          pressed: button.pressed,
-        };
-      }
-    );
-
-    // TODO: swpa this out for `ExtensionProvider.getExtension()`
-    // once IDendronExtension interface has the tree view properties.
-    const lookupView = getExtension().getTreeView(
+    const lookupView = ExtensionProvider.getTreeView(
       DendronTreeViewKey.LOOKUP_VIEW
     ) as LookupView;
-    lookupView.refresh(payload);
+    lookupView.refreshLookupView({ buttons: opts.quickpick.buttons });
```

Comment:
```suggestion
    lookupView.refresh({ buttons: opts.quickpick.buttons });
```

See suggestion in `LookupView.ts`